<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreviewImage</class>
 <widget class="QWidget" name="PreviewImage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1188</width>
    <height>756</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="CamName">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>1071</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Computer #1, Username</string>
   </property>
  </widget>
  <widget class="QLabel" name="OriginImage">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>571</width>
     <height>411</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Origin Image #1</string>
   </property>
  </widget>
  <widget class="QLabel" name="CameraImage">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>180</y>
     <width>551</width>
     <height>411</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Image from camera</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>610</y>
     <width>421</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Similarity : </string>
   </property>
  </widget>
  <widget class="QPushButton" name="prev">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>690</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Prev</string>
   </property>
  </widget>
  <widget class="QPushButton" name="next">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>690</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startQuiz">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>660</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Start Quiz</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopQuiz">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>660</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Stop Quiz</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadImage">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>660</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>LoadImage</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ip">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>600</y>
     <width>501</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Enter username here </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
